src/components/LabEditorGUI/ContextMenu.tsx

```diff
@@ -0,0 +1,58 @@
+import React, {useEffect, useState} from 'react';
+import './ContextMenu.css';
+
+const ContextMenu = ({parentRef, items}:any) => {
```

Comment:
avoid use any, go through and find usages of any and find what type they should be.